.github/workflows/manual-deploy-docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Docs deploy to Netlify
+on:
+  workflow_dispatch:
+    inputs:
+      docsVersion:
+        description: "Version of the Docs to release. Must be in format: vX eg.: v6 v7 ..."
+        required: true
+        default: "v7"
```

Comment:
shouldn't this be "v6" on this branch?

Reply:
Doesn't really matter, because when you run this workflow on GitHub it will use the default value from the master workflows file.